ringpop.go

```diff
@@ -226,12 +258,23 @@ type BootstrapOptions struct {
 
 // Bootstrap starts the Ringpop
 func (rp *Ringpop) Bootstrap(opts *BootstrapOptions) ([]string, error) {
+
```

Comment:
+1 to no blank line at the beginning or end of a function. The only time I think I'd add a blank line is if you have a lot of arguments split over multiple lines, e.g.
```go

func(arg1 *somepackage.ReallyBigType, arg2 *somepackage.AnotherBigType, arg2 *somepackage.ReallyBigType,
    arg4 *somepackage.AnotherType, arg5 *somepackage.TooManyArgs) {

  // code here
}